packages/engine/Source/Shaders/Builtin/Functions/translateRelativeToEye.glsl

```diff
@@ -34,6 +34,7 @@
 vec4 czm_translateRelativeToEye(vec3 high, vec3 low)
 {
     vec3 highDifference = high - czm_encodedCameraPositionMCHigh;
+    if (length(highDifference) == 0.0) highDifference = vec3(0);
```

Comment:
Based on our [Code Style Guide](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md) this should be formatted as
```suggestion
    if (length(highDifference) == 0.0) {
        highDifference = vec3(0);
    }
```